pkg/chartutil/coalesce.go

```diff
@@ -175,8 +175,14 @@ func coalesceValues(c *chart.Chart, v map[string]interface{}) {
 //
 // dest is considered authoritative.
 func CoalesceTables(dst, src map[string]interface{}) map[string]interface{} {
+	// When --reuse-values is set but there are no modifications yet, return new values
 	if dst == nil || src == nil {
```

Comment:
Can the external `if dst == nil || src == nil` just be removed? I think your patch addresses both halves of the `or`.

Reply:
Sure thing